provider/requirements.txt

```diff
@@ -0,0 +1,46 @@
+anyio==3.3.4
+appdirs==1.4.4
+asgiref==3.4.1
+attrs==21.2.0
+certifi==2020.12.5
+charset-normalizer==2.0.7
+click==8.0.3
+distlib==0.3.1
+dnspython==2.1.0
+email-validator==1.1.3
+fastapi==0.70.0
+filelock==3.0.12
+h11==0.12.0
+httptools==0.2.0
+idna==3.3
+iniconfig==1.1.1
+itsdangerous==2.0.1
+Jinja2==3.0.2
+MarkupSafe==2.0.1
+orjson==3.6.4
+packaging==21.2
+pbr==5.5.1
+pluggy==1.0.0
+py==1.11.0
+pydantic==1.8.2
+pyparsing==2.4.7
+pytest==6.2.5
+python-dotenv==0.19.1
+python-multipart==0.0.5
+pywatchman==1.4.1
+PyYAML==5.4.1
+requests==2.26.0
+six==1.15.0
+sniffio==1.2.0
+starlette==0.16.0
+stevedore==3.3.0
+toml==0.10.2
+typing-extensions==3.10.0.2
+ujson==4.2.0
+urllib3==1.26.7
+uvicorn==0.15.0
+uvloop==0.16.0
```

Comment:
uvloop install from requirements.txt not for Windows OS

Reply:
@amecklenborg Yes indeed, this is intended for the container which has unix based OS based and Python:3.9.7 installed. Assuming you have docker install, it should still work on your machine. however, outside the container, I can not guarantee it can run every machine ¯\_(ツ)_/¯